SharpBucket/V2/EndPoints/RepositoriesEndPoint.cs

```diff
@@ -15,6 +15,17 @@ namespace SharpBucket.V2.EndPoints
     /// </summary>
     public class RepositoriesEndPoint : EndPoint
     {
+        private class FilterSortDictionary : Dictionary<string, object>
```

Comment:
I'm not convinced by the idea of creating a new type for that
I think that having just a helper method that return an IDictionary should be better and may lead to less memory allocation since we may return null instead of an empty instance

Something like:
```CSharp
private IDictionary<string, object> BuildFilterSortParameters(string, filter, string sort)
{
    IDictionary<string, object>parameters;
    if (!String.IsNullOrWhiteSpace(filter))
        (parameters ?? (parameters = new Dictionary<string, object>()))["q"] = filter;
    if (!String.IsNullOrWhiteSpace(sort))
        (parameters ?? (parameters = new Dictionary<string, object>()))["sort"] = sort;
}
```

using it should then looks to
```CSharp
var dict = BuildFilterSortParameters(filter, sort);
```
Instead of
```CSharp
var dict = new FilterSortDictionary(filter, sort);
```

Reply:
Sure, I'll make this change.